src/tests/tablet/home/AvailableMessageDisplayedWhenRoomIsFreeAfterAMeeting.java

```diff
@@ -0,0 +1,59 @@
+package tests.tablet.home;
```

Comment:
We don't need to upload the test cases yet, we need the class of the framework as soon as possible and as a team we according that.